ganttproject/src/main/java/net/sourceforge/ganttproject/storage/ProjectDatabase.kt

```diff
@@ -43,6 +51,9 @@ interface ProjectDatabase {
     @Throws(ProjectDatabaseException::class)
     fun commit()
 
+    fun setName(oldName: String?, newName: String?)
```

Comment:
I think we should give up MutableTask interface rather that having two TaskUpdateBuilders for ordinary commit and undo